.test-infra/jenkins/common_job_properties.groovy

```diff
@@ -147,17 +150,20 @@ class common_job_properties {
         if (onlyTriggerPhraseToggle) {
           onlyTriggerPhrase()
         }
+        if (!triggerPathPatterns.isEmpty()) {
+          includedRegions(triggerPathPatterns.join('\n'))
```

Comment:
The docs imply that `includedRegions` should accept varargs of strings, not newline separated. Did you determine that this was really how it works?

Reply:
Hmm, well I was looking at autogenerated docs for the SVN config, as it happens.

Reply:
You can view the documentation via the Job DSL API viewer [here](https://builds.apache.org/plugin/job-dsl/api-viewer/index.html#method/org.jenkinsci.plugins.ghprb.GhprbTrigger.includedRegions):

> Each inclusion uses regular expression pattern matching, and must be separated by a new line. An empty list implies that everything is included.